src/ts/services/adapters/TyperighterChunkedAdapter.ts

```diff
@@ -37,7 +37,8 @@ class TyperighterChunkedAdapter extends TyperighterAdapter
       }),
       body: JSON.stringify({
         requestId,
-        blocks
+        blocks,
+        categoryIds
```

Comment:
It looks like we're sending `categoryIds` twice, once at the root level, once inside `blocks`, was that intended?

![Screenshot 2023-09-05 at 15 00 41](https://github.com/guardian/prosemirror-typerighter/assets/7423751/7f99d767-6ad7-4659-adf4-4beced54db51)